src/components/address/AddressLookup/__tests__/AddressLookup-test.js

```diff
@@ -1,17 +1,13 @@
-'use strict';
-jest.autoMockOff();
-
+jest.disableAutomock();
```

Comment:
Switching from `jest.autoMockOff()` to `jest.disableAutomock()` is done to work nice with ES6 imports that get hoisted.

See: https://github.com/facebook/jest/issues/377 for more detail.